cinema/serializers.py

```diff
@@ -79,8 +99,16 @@ class Meta:
             "cinema_hall_name",
             "cinema_hall_capacity",
             "tickets_available",
+            "movie_image"
         )
 
+    def get_movie_image(self, obj):
+        movie = obj.movie
+        if movie.image:
+            return movie.image
+        else:
+            return None  #
```

Comment:
The `get_movie_image` method should return the URL of the image rather than the image field itself. Use `movie.image.url` to get the URL if the image exists.